Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -1062,7 +1055,8 @@ recentSearchesView state push =
     , height WRAP_CONTENT
     , orientation VERTICAL
     , margin $ MarginTop 16
-    , visibility if null state.data.destinationSuggestions then GONE else VISIBLE
+    , padding $ PaddingHorizontal 16 16
+    , visibility $ boolToVisibility $ state.data.destinationSuggestions /= []
```

Comment:
not null can we used instead of /= []

Reply:
`not null` will do two operations where as `/=` this will do single operation

Reply:
will change.